apps/src/lib/levelbuilder/lesson-editor/ActivitySectionCard.jsx

```diff
@@ -303,19 +302,19 @@ class ActivitySectionCard extends Component {
         id: NEW_LEVEL_ID,
```

Comment:
The level has an id because it already has been create but the script level has not been created. So we put this in to hold the place until we create the new script level.